pom.xml

```diff
@@ -66,6 +66,12 @@
 			<version>4.19</version>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.avro</groupId>
+			<artifactId>avro</artifactId>
+			<version>1.11.1</version>
```

Comment:
This resolved the detected issue; however, it will eventually need to be removed once we upgrade dropwizard-metrics (which will be required eventually but that can be a different effort for now).